index.html

```diff
@@ -7,7 +7,7 @@
 
     <title>Welcome to Potionfy</title>
   </head>
-  <h1>Welcome to Potionfy! Enjoy your stay</h1>>
+  <h1>Welcome to Potionfy!</h1>>
```

Comment:
Could you remove the 'enjoy your stay' string and make the 'Welcome to Potionfy!' string red?